src/Elasticsearch/Connections/Connection.php

```diff
@@ -574,6 +579,26 @@ protected function getCurlRetryException(array $request, array $response): Elast
         return $exception;
     }
 
+    /**
+     * Get the x-elastic-client-meta header
+     */
+    private function getElasticMetaHeader(array $connectionParams): string
+    {
+        $clientMeta = sprintf(
+            "es=%s,php=%s,a=%d",
```

Comment:
Need to add `t=...` for PHP transport version is the same as client version I assume?

Reply:
I didn't insert the transport `t` parameter because I read this in the specification: `"If there is no transport library separate from the client library then the key must be omitted"`.

Reply:
I added the `t` parameter since the spec was outdated.